src/mechanisms.rs

```diff
@@ -47,7 +47,8 @@ pub struct P256Prehashed {}
 mod p256;
 
 pub struct Rsa2kPkcs {}
-mod rsa2kpkcs;
+// Later on we'll add: "pub struct Rsa2kPss {}" and so on
+mod rsa2k;
```

Comment:
Looks like you forgot to `git add` the new `src/mechanisms/rsa2.rs` after renaming.